internal/service/timestreaminfluxdb/db_instance.go

```diff
@@ -310,15 +209,26 @@ func (r *resourceDbInstance) Schema(ctx context.Context, req resource.SchemaRequ
 						"",
 					),
 				},
+				Description: `The name that uniquely identifies the DB instance when interacting with the 
+					Amazon Timestream for InfluxDB API and CLI commands. This name will also be a 
+					prefix included in the endpoint. DB instance names must be unique per customer 
+					and per region.`,
 			},
 			names.AttrTags:    tftags.TagsAttribute(),
 			names.AttrTagsAll: tftags.TagsAttributeComputedOnly(),
 			"organization": schema.StringAttribute{
```

Comment:
When running `terraform apply` with the configuration you noted [here](https://github.com/Bit-Quill/terraform-provider-aws/pull/2#issuecomment-2134094671), I got the following error:


```
Error: Provider produced inconsistent result after apply

When applying changes to aws_timestreaminfluxdb_db_instance.test_db_instance, provider "provider[\"registry.terraform.io/hashicorp/aws\"]" produced an unexpected new value:
.organization: was null, but now cty.StringVal("organization").

This is a bug in the provider, which should be reported in the provider's own issue tracker.
```

The instance still deployed but I'm not sure what caused this error.

Reply:
Item was resolved after recompile.